lms/templates/learner_dashboard/course_card_actions.underscore

```diff
@@ -0,0 +1,11 @@
+<% if(enrollment_status){ %>
+    <div class="enrollment-info"><%- gettext('ENROLLED') %></div>
+    <a id="view_course_link" href="<%- courseware_url %>" class="btn-neutral btn view-course-link">
```

Comment:
ids should be unique, for this all course cards will have the same id. Why do we need an id here at all?